django/api/serializers/user.py

```diff
@@ -0,0 +1,39 @@
+"""
+Further reading:
+https://www.django-rest-framework.org/api-guide/serializers/
+"""
+from rest_framework.serializers import ModelSerializer, SerializerMethodField
+
+from api.models.user import User
+from api.models.user_permission import UserPermission
+from api.serializers.permission import PermissionSerializer
+
+class UserSerializer(ModelSerializer):
+    """
+    Default Serializer for User
+    """
+    user_permissions = SerializerMethodField()
+    idir = SerializerMethodField()
```

Comment:
I don’t think we need to use a SerializerMethodField for idir, since idir is a CharField and we’re serializing user objects (so just including “idir” in the “fields” attribute under “Meta” should be enough)